packages/contracts/test/framework/dao/dao-factory.ts

```diff
@@ -551,16 +534,21 @@ describe('DAOFactory: ', function () {
 
     const plugins = [plugin1, plugin2];
     const tx = await daoFactory.createDao(daoSettings, plugins);
-    const {events} = await tx.wait();
 
-    let installEventCount = 0;
+    // Count how often the event was emitted by inspecting the logs
+    const receipt = await tx.wait();
+    const topic = PluginSetupProcessor__factory.createInterface().getEventTopic(
+      EVENTS.InstallationApplied
+    );
+
+    let installationAppliedEventCount = 0;
 
     // @ts-ignore
-    events.forEach(event => {
-      if (event.event == EVENTS.InstallationApplied) installEventCount++;
+    receipt.logs.forEach(log => {
+      if (log.topics[0] == topic) installationAppliedEventCount++;
```

Comment:
I suggest you use strict comparison, because type juggling and hex strings could convert sometimes to numbers.

Reply:
even it is more of a problem in PHP it still maybe could lead to issues

Reply:
Fixed in 306579de100b535c3dee806707b6b7c697206875